src/tir/schedule/primitive/cache_read_write.cc

```diff
@@ -94,6 +94,132 @@ Optional<BufferRegion> GetBufferRegionFromBuffer(const Array<BufferRegion>& buff
   return res;
 }
 
+struct ReindexCacheStageInfo : CacheStageInfo {
+  /* Indices used to access the allocated cache buffer. */
+  Array<PrimExpr> indices;
+  /* Touched loop variable related information. */
+  Array<Var> loop_vars;
+  Array<Range> loop_ranges;
+  /* Touched block variable related information. */
+  Array<IterVar> block_iter_vars;
+  Array<PrimExpr> block_iter_values;
+};
+
+/* \brief The schedule error that accessed buffer region is not a single point for
+ * reindex_cache_read/write. */
```

Comment:
```suggestion
/* 
 * \brief The schedule error that accessed buffer region is not a single point for
 * reindex_cache_read/write. 
 */
```
Please use multi-line style here